portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase

```diff
@@ -212,6 +212,64 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-150460 TC-1. User is able to trigger propagation of the site template to all connected sites automaticly."
```

Comment:
"automaticly" I'm not sure if this is an existing word.

Reply:
Changed to: automatically